### ISSUE TITLE
Adding to the `render(tagName, importer)` function an optional third parameter for initial attributes

### ISSUE THREAD

**Issue by mesr** [ficusjs/ficusjs-testing#17]
Hello there! And thank you for putting together and maintaining this absolutely awesome framework!

I am writing tests for components that define required attributes (props) with no default values. Using the provided `render(tagName, importer)` function renders components invariably without attributes. Of course, I can set those attributes on the component immediately after rendering, but then the initial rendering still prints various warning messages that don't look nice in the test run and, in several places, actually makes the test fail because some underlying function used by the component's `render()` throws an error when no value is supplied (which is how the component is designed and expected to behave in such case).

I am easily working around this limitation with a modified version of the `render(tagName, importer)` function:

```javascript
export async function render (tagName, importer, attrs = {}) {
  await importer()
  const document = globalThis.document
  const basicComp = document.createElement(tagName)
  Object.entries(attrs).forEach(([ name, value ]) => {
    basicComp.setAttribute(name, value)
  })
  document.body.appendChild(basicComp)
  return basicComp
}
```

Yet, it would be nice imho to see this small improvement make its way into the *ficusjs-testing* package. Or is there already a recommended way of achieving this that I would have missed? What are your thoughts on that?

**Comment by ducksoupdev** [comment]
Hi @mesr, thanks for pointing this out. Your suggestion would solve the issue. I'll get this change added and release a new version.

**Comment by ducksoupdev** [comment]
This has been released in v2.2.0 - https://github.com/ficusjs/ficusjs-testing/releases/tag/2.2.0
